tests/integration/components/self-clear-cache-test.js

```diff
@@ -0,0 +1,94 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+// Lets say we have component with name 'self-clear-cache.hbs'
+// The UI Design can be found here : https://www.figma.com/file/LMujWCqnzXl1wQJVwAzCPV/Clear-your-member-details-page?node-id=0%3A1
+// It shows the time at which user clears the cache last time.
+// Last Requested : {{ time }}
+
+module('Cache Clear Page', (hooks) => {
```

Comment:
Let's follow the standard way of writing the module names
`Integration | Component | self-clear-cache`

Then, we can simply run the test suite: `npx ember test --server --filter="Integration | Component | self-clear-cache"`